src/SFA.DAS.AssessorService.Application.Api.External/Controllers/StandardsController.cs

```diff
@@ -66,5 +66,27 @@ public async Task<IActionResult> GetOptionsForStandard([SwaggerParameter("Standa
 
             return Ok(requestedStandard);
         }
+
+        [HttpGet("options/{standardReference}/{version}")]
```

Comment:
Just ran this locally, the url doesn't adhere to the other endpoints, should be /api/v1/standards/options/{standardref}/{version}